configure.ac

```diff
@@ -15,9 +15,29 @@ AC_PROG_INSTALL
 AC_PROG_RANLIB
 AC_PROG_MAKE_SET
 RC_IF_INSTALL_OPTS
+builddirs="lib spmd"
 
-AC_CONFIG_SUBDIRS([lib spmd iked kinkd])
+AC_MSG_CHECKING(if --enable-kinkd option is specified)
+AC_ARG_ENABLE(kinkd,
+    [ --enable-kinkd           enable kinkd[yes]],
+    AC_MSG_RESULT($enable_kinkd), 
+    AC_MSG_RESULT(default to yes)
+    enable_kinkd=yes
+    ac_configure_args="$ac_configure_args --enable-kinkd")
 
+if test "x$enable_kinkd" = xyes; then 
+   builddirs="$builddirs kinkd"
+fi
+AC_MSG_CHECKING(if --enable-iked option is specified)
+AC_ARG_ENABLE(iked, [  --enable-iked           build iked [yes]],
```

Comment:
Thanks! Why are the two options formatted slightly differently, and why don't we add ac_configure_args to the iked part?

Reply:
I wrote the first one, then I noticed the option that was previously removed would work just needed to use the same variable. I can reformat them the same if you would like.

Reply:
Yes, just make them look consistent either way you want, so people don't wonder why they look different.